modules/libcom/src/osi/os/WIN32/osdTimePvt.h

```diff
@@ -1,21 +1,24 @@
 /*************************************************************************\
-* Copyright (c) 2011 UChicago Argonne LLC, as Operator of Argonne
+* Copyright (c) 2002 The University of Chicago, as Operator of Argonne
```

Comment:
This seems an accidental change...

Reply:
It seems to be a git/github artifact. This is a new file that was created as a copy+edit from osdTime.h (which has the 2002 copyright header) and added to this branch. I don't know how this diff was created. If I run

       `git diff 7.0..HEAD`

the output looks very different (new file).

Nevertheless, thanks to your comment I found a typo in a comment in this very file, thanks ;-)